app/src/androidTest/java/io/github/omisie11/spacexfollower/NavDrawerEspressoTests.kt

```diff
@@ -17,7 +17,6 @@ import io.github.omisie11.spacexfollower.ui.MainActivity
 import org.junit.Rule
 import org.junit.Test
 import org.junit.runner.RunWith
-
```

Comment:
This line shouldn't be deleted, I think that blank line between imports and file content should be here

Reply:
okay, should I add blank ??

Reply:
Yes, You deleted it in Your commit 👍